nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattClientNotificationResult.cs

```diff
@@ -0,0 +1,47 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.Device.Bluetooth.GenericAttributeProfile
+{
+    /// <summary>
+    /// The result of NotifyValue
+    /// </summary>
+    public sealed class GattClientNotificationResult
```

Comment:
Mainly because the original UWP class was sealed. 
Don't see any reason why you would want to derive from this class.
What do you think. Should i remove it ?

Reply:
This applies to all sealed classes you have mentioned.

Reply:
if you have no good reason to sealed them, then remove this. It doesn't really bring any value at all and will give flexibility if needed.

Reply:
Removed